docs/pages/index.mdx

```diff
@@ -2,39 +2,134 @@ import ImageRow from "@/components/ImageRow";
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-origin-registration.png"}>
+	When the Origin starts, it's told which federation it belongs to. It discovers the
+	hostnames for its Registry and Directory by using the federation root to construct
+	the URL "https://osg-htc.org/.well-known/pelican-configuration", which contains a JSON
+	detailing the federation's central services.
+
+	Next, the Origin registers its namespace and public key with the Registry, proving that it
+	owns the corresponding private key. Finally, the Origin begins advertising its namespace
+	information and hostname to the Director.
+</ImageRow>
+
+While somewhat simplified, this example illustrates the process origins must take to make themselves known within the federation. After completing these steps, the objects from the Data Repository are available via Pelican.
+
+The next step is for the data consumer to actually *move* the data. Pelican assumes the data consumer already knows the federation that provides the data they want, along with the name of the object within the federation. These two pieces of information are combined and provided to the Client as a `pelican://`-schemed URL
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-orig-discovery.png"}>
+	The data consumer provides their Pelican client of choice the pelican:// URL that defines
+	the object they want to download, where osg-htc.org is the federation and
+	(/weather/cloud.jpg) is the object. Just as the origin discovered the Director's hostname
+	by visiting "https://osg-htc.org/.well-known/pelican-configuration", so too does the client.
+
+	After the client has performed federation metadata discovery, it issues an HTTP GET
+	request to the Director, using the object name as a URL path. The Director responds
+	with an HTTP 307 Redirect, forwarding the client on to the a server that can provide
+	the object, in this example an Origin.
+
+	Finally, the Client follows the redirect and downloads the object by issuing an HTTP
+	GET request to "https://my-origin.com/weather/cloud.jpg"
+
+	Notice that the Origin continues advertising with the Director throughout.
+</ImageRow>
 
-Central to Pelican is the concept of the origin service.  The origin is the intermediary between
-the existing storage and the federation.  The origin is responsible for serving data as well
-as issuing tokens (credentials) authorizing access to datasets based on the local policy.
+Once again, this example is simplified, mainly because the Director typically sends the client to a Staging device capable of fetching the object, not directly to the Origin. In any case, the object is delivered to the Client without passing through the federation's Central Services. When the object is fetched through a Staging device, the Staging device performs the same discovery step as the Client by asking the Director for an Origin that exports the object.
```

Comment:
I see...but I'm not sure if we want to skip caches completely, which really depends on who our target users are for this overview..

Reply:
My thinking is that the process the cache goes through to find an origin is the same as the client goes through to find an origin (bar a few gritty details that live in xrdcl-pelican). From the perspective of "how does a client discover a service that can provide data", there's essentially no difference between the cache and the origin, and throwing in the cache right away adds an extra layer of complexity. I'm personally in favor of leaving it out in this example, but if enough other people disagree, I'm happy to add it. @aowen-uwmad do you think adding the cache layer to the middle of this and getting into those details would help users or confuse them more?

Reply:
@jhiemstrawisc , I think the proposed change is fine, I think the reference to the "Staging device" (which was also defined earlier in the terminology section) is enough for this level of detail. If people want more info on how the Cache comes into play, they can read up the documentation about the Cache.